packages/components/components/elvis-pagination/CHANGELOG.json

```diff
@@ -2,6 +2,16 @@
   "$schema": "../../changelogSchema.json",
   "name": "elvis-pagination",
   "content": [
+    {
+      "date": "12.09.24",
+      "version": "3.6.0",
+      "changelog": [
+        {
+          "type": "new_feature",
+          "changes": ["Added aria label support for switching language."]
```

Comment:
Perhaps mention that it happens automatically based on the `lang` attribute, and that it's not a new prop people need to consider

Reply:
Could this work?
`"Added aria-label support for switching language automatically based on current lang attribute value."`